### PR TITLE
features:added routing and also toggle sidebar

### PR DESCRIPTION
**Pull request by weberankit**
what I have done:-
added routing and arrange component

toggle sidebar

https://github.com/0silverback0/TrainerAppFrontend/assets/94105514/45ddaedd-2eb2-4fb6-bc91-cc426f2d81c2

### CODE REVIEW

**Comment by weberankit** [comment]
I have also changed the arrangement of component a/c to routing following best practices and it is the most convenient and best way to do...  solved  issue #7

**Comment by 0silverback0** [comment]
im getting some redirect issues i think its a netlify issue honestly because ive had this problem before using router on netlify I want to see if this can be fixed before i merge

**Comment by weberankit** [comment]
okay

**Comment by 0silverback0** [comment]
have you encountered an error before using react router on netlify?

**Comment by weberankit** [comment]
@0silverback0 no could you plz send the screenshot of the error or tell me more about an error I will try to fix it

File: src/App.js [hunk 1]
```diff
@@ -1,20 +1,60 @@
 import './App.css';
-import CoachInfo from './CoachInfo';
-import ExerciseSearch from './ExerciseSearch';
+//import CoachInfo from './CoachInfo';
+//import ExerciseSearch from './ExerciseSearch';
 import Header from './Header';
```
Comment (0silverback0) [inline src/App.js:4]: its here header is defined but never used is the error. this may or may not fix the issue that just the current error

**Review by 0silverback0 — CHANGES_REQUESTED** [review]

**Comment by weberankit** [comment]
@0silverback0  plz check it (Header import deleted)

**Comment by weberankit** [comment]
@0silverback0 Error is fixed How?
checkout this link https://githubsite.netlify.app/
what I have done is 
removed the build from gitignore file and run the command npm run build and I have manually deployed that build file on Netlify 

if you allow me then I will make pr but plz note that pr contains the build file also 
image

**Comment by weberankit** [comment]
or should I make PR by removing the build file as you like I will do it.

**Comment by 0silverback0** [comment]
nope that works thank you so much!

**Pull request merged**